### ISSUE TITLE
Question about multiscale_training

### ISSUE THREAD

**Issue by CCodie** [eriklindernoren/PyTorch-YOLOv3#725]
Thanks for sharing this awesome code.

And I have a question about multiscale_training.

YOLOv3 does support multiscale_training because it is consist of Fully Convolutional Network.
And it seems like it is implemented at function `collate_fn` in `utils/datasets.py` file as below.

      def collate_fn(self, batch):
          self.batch_count += 1
  
          # Drop invalid images
          batch = [data for data in batch if data is not None]
  
          paths, imgs, bb_targets = list(zip(*batch))
  
          # Selects new image size every tenth batch
          if self.multiscale and self.batch_count % 10 == 0:
              self.img_size = random.choice(
                  range(self.min_size, self.max_size + 1, 32))
  
          # Resize images to input shape
          imgs = torch.stack([resize(img, self.img_size) for img in imgs])
  
          # Add sample index to targets
          for i, boxes in enumerate(bb_targets):
              boxes[:, 0] = i
          bb_targets = torch.cat(bb_targets, 0)
  
          return paths, imgs, bb_targets

I can see the image resizing part, but can you tell me **where's resizing target boxes** ?
What I think is that if image samples are resized, labeled boxes also need to be resized accordingly.

Thanks !

**Comment by Flova** [comment]
Hi,

thanks for reporting this issue and helping the project. 
The boxes are in a relative coordinate system at this point in the pipeline, meaning they are described as a fraction of the width and height of the image.
This had also been discussed in #721

Cheers

**Comment by Flova** [comment]
Exactly, but they are converted to absolute labeles and back for data augmentation, so technically it's not due to the label format, but the label format is also resolution independent.
